### ISSUE TITLE
Put List of Specific AWS Services in Alpha Order

### ISSUE THREAD

**Issue by rjhintz** [open-guides/og-aws#304]
The list is long enough that it's a pain to scan by eye. It's not as long as it will be once we include other services such as SQS.

**Comment by lynnlangit** [comment]
We've had some discussion on this already.  There are a number of different approaches we could take to organizing the list of AWS services. We would be interested in the opinion of our readers on the organization.

Currently we organize by **frequency of usage**, i.e. things like IAM, S3 and EC2 first, as they are the most used services.

We could organize **alphabetically by service name**, or we could follow a **categorical organization** scheme - such as following the AWS Console, or organizing by service type (i.e. compute, data, networking, etc...)

We really want to make the og-aws SCANNABLE, so feel free to weigh in here and/or in our Slack channel as to what works best for you.

**Comment by jlevy** [comment]
Yes. Alphabetically across all services seems unhelpful to me, since very important services get lost within the long list. (Same issue with the AWS console.) I think the right answer longer term probably involves a hybrid of grouping and categories.

We could also align it with the categories here, which currently starts with "must-know infra" and has other large grups.
https://github.com/open-guides/og-aws#which-services-to-use

**Comment by jlevy** [comment]
If anyone has thoughts/feedback on the big buckets at https://github.com/open-guides/og-aws#which-services-to-use
do put down your thoughts here too.

**Comment by rjhintz** [comment]
The advantage to alpha order is that it's easily understood and avoids groups of only 1 member, such as IoT. 

Names for any ordering are a bit of a puzzle, too. For absolute newcomers, there seems to be a need to decode "ec2" to "compute." Types of storage are a minefield, too. "S3" doesn't exactly shout "object storage."

The disadvantage to grouping as families of services is that the grouping can be request some prior understanding, as with SNS under mobile in the AWS scheme. Of course AWS uses alpha ordering in one place and by groups in another to get around some of these issues.

I can understand opposing views and I'll be interested to hear other opinions. I'm just approaching from the requirement that I want to add "SQS" and would prefer not having two weeks discussion to figure out where to put it. Since I'm a late comer, though, I'm inclined to go with the committers' POV.

**Comment by jlevy** [comment]
This issue should not block any new sections. @rjhintz if you begin a SQS section, we should review/discuss in Slack+Quip as usual to get to first draft state. But for positioning let's put it right in front of Kinesis, since it's probably similar in popularity and type of service. Thanks!

**Comment by bgdnlp** [comment]
FWIW, I'd go with alphabetical. I'm thinking the most common way of using the guide, after the first time, would be to go to the list of services, click the service you're interested in and read that section, instead of reading the whole thing. Alphabetical order would be practical in this case.

Plus, it avoids discussion on what should go where. For example, I don't think that SQS should go with Kinesis. SQS is not data, it's a way to do application orchestration, and I doubt it is as popular simply because of that.

Also, there is no data on how popular services are AFAIK and they can rise and fall in popularity, especially when Amazon will want to phase one of them out or focus on improving another.

I'd rather see the ToC expanded and maybe have a section on how popular services are with links. If anyone is interested in that way of looking at AWS, they can use that section.
